src/main/java/appeng/core/Registration.java

```diff
@@ -122,16 +124,11 @@
 import appeng.worldgen.QuartzWorldGen;
 
 
-public final class Registration
+final class Registration
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a constructor to the class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1258)